File: botSpotifyV1/crearUsuarios.py
import math
from random import random
from time import time

def getRandomName():
    n = ["b", "c", "d", "f", "g", "h", "j", "k", "l", "m", "n", "p", "r", "s", "t", "v", "w", "x", "y", "z"]
    t = ["a", "e", "i", "o", "u"]
    r = getRandomInt(5, 7, True)
    o = ""
    i = True
    while len(o) < r:
        if i:
            o += str(n[getRandomInt(0, len(n) - 1)])
            i = False
        else:
            o += str(t[getRandomInt(0, len(t) - 1)])
            i = True
    a = 10 - r
    tiempo = str(time()).split('.')
    #return o + str(getRandomInt(math.pow(10, a - 1), math.pow(10, a)))
    return F"{o}{tiempo[0]}{tiempo[1][:3]}"


def getRandomInt(n, t, r=False):
    o = 1 if r else 0
    return int(math.floor(random() * (t - n + o)) + n)

File: botSpotifyV1/crearUsuarios.js
function getRandomName() {
    let n = ["b", "c", "d", "f", "g", "h", "j", "k", "l", "m", "n", "p", "r", "s", "t", "v", "w", "x", "y", "z"],
        t = ["a", "e", "i", "o", "u"],
        r = getRandomInt(5, 7, true),
        o = "",
        i = true;
    for (; o.length < r;) i ? (o += n[getRandomInt(0, n.length - 1)], i = false) : (o += t[getRandomInt(0, t.length - 1)], i = true);
    const a = 10 - r;
    return o + getRandomInt(Math.pow(10, a - 1), Math.pow(10, a))
}
function getRandomInt(n, t, r = false) {
    const o = r ? 1 : 0;
    return Math.floor(Math.random() * (t - n + o)) + n
}

getRandomName()